internal/contour/secret.go

```diff
@@ -107,6 +107,10 @@ func (v *secretVisitor) visit(vertex dag.Vertex) {
 		if svh.FallbackCertificate != nil {
 			v.addSecret(svh.FallbackCertificate)
 		}
+	case *dag.Cluster:
+		if svh.ClientCertificate != nil {
```

Comment:
nit: let's rename `svh` now that it's not only for a `SecureVirtualHost` -- `obj` or something similarly generic.

Reply:
Fixed!